users/models.py

```diff
@@ -161,3 +164,9 @@ class OneTimePassword(models.Model):
 
     class Meta:
         db_table = "one_time_password"
+
+
+@receiver(post_save, sender=User)
+@receiver(post_delete, sender=User)
+def user_update_cache(sender, instance, created, raw, **kwargs):
```

Comment:
```suggestion
def user_update_cache(_, instance, _, _, **kwargs):
```

Reply:
This didn't work and started throwing duplicate parameter `_`. Can you clarify why you suggested this change?